src/qt/guiutil.cpp

```diff
@@ -879,4 +884,22 @@ int TextWidth(const QFontMetrics& fm, const QString& text)
 #endif
 }
 
+void LogQtInfo()
+{
+#ifdef QT_STATIC
```

Comment:
nit, do
```
#if defined(QT_STATIC) || defined(QT_STATICPLUGIN)
```
and remove the two conditions below

Reply:
qt_link and plugin_link are orthogonal.